### PR TITLE
New Edits for Review

### PR DESCRIPTION
**Pull request by ehteshama12**
Please review the new edit.

### CODE REVIEW

File: src/team/g6/DemoG6.java [hunk 1]
```diff
@@ -63,6 +63,8 @@ public static void addition(int c,int d) { //REPOSITORY
 	public static void goodBye() { //LOCAL
 		String goodBye;
 			// just another test
-		
+
```
Comment (AshleeValle) [inline src/team/g6/DemoG6.java:66]: great!

**Review by AshleeValle — APPROVED** [review]

**Comment by Gibtsawit** [comment]
Reviewed!

**Comment by Surfsup1408** [comment]
Needs a little more Spice Eht, but I'll accept, good job!

**Comment by ehteshama12** [comment]
> Needs a little more Spice Eht, but I'll accept, good job!

I'll work on it a bit more then! Thank you for the advice!

**Review by Gibtsawit — COMMENTED** [review]
Reviewed

**Review by Surfsup1408 — COMMENTED** [review]
Eht did it!

**Pull request merged**